docker/python/Dockerfile

```diff
@@ -0,0 +1,38 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM ubuntu:18.04
+
+ENV DEBIAN_FRONTEND noninteractive
+
+# http://bugs.python.org/issue19846
+# At the moment, setting "LANG=C" on a Linux system fundamentally breaks
+# Python 3.
+ENV LANG C.UTF-8
+
+# Install dependencies.
+RUN apt-get update \
+  && apt-get install -y --no-install-recommends \
+  python3 \
+  python3-distutils \
+  sudo \
+  wget \
+  && apt-get clean autoclean \
```

Comment:
I'm used to seeing multiple statements like this be multiple RUN statements.  Is there a reason to prefer &&ing them together?

Reply:
Ok.  I read https://docs.docker.com/develop/develop-images/dockerfile_best-practices/, and that explained it.